.github/workflows/reusable_image_release.yaml

```diff
@@ -0,0 +1,251 @@
+name: Reusable Pre Release and Release Pipeline for docker images
+
+on:
+  workflow_call:
+    inputs:
+      original_repo_name:
+        description: 'Full repository name (org/repo)'
+        type: string
+        required: true
+      docker_platforms:
+        description: 'Comma-separated list of Docker platforms'
+        type: string
+        default: 'linux/amd64,linux/arm64'
+      docker_image_name:
+        description: 'Docker image name to use'
+        type: string
+        required: true
+      integration_name:
+        type: string
+        required: false
+        default: ''
+      s_three_path:
+        type: string
+        required: false
+        default: ''
+      s_three_folder:
+        type: string
+        required: false
+        default: ''
+      go_version_file:
+        type: string
+        required: false
+        default: 'go.mod'
+      release_command:
+        type: string
+        required: true
+
+      run_integration_tests:
+        type: boolean
+        required: false
+        default: false
+      run_nix_unit_tests:
+        type: boolean
+        required: false
+        default: true
+      run_windows_unit_tests:
+        type: boolean
+        required: false
+        default: true
+      use_build_push_action:
```

Comment:
I checked 3 PRs and all of them had use_build_push_action=false and use_custom_release=true.

Should we make having a custom command a "must"?